### PR TITLE
wg-multitenancy: add working group

### PR DESCRIPTION
**Pull request by jessfraz**
cc @davidopp @ericchiang :)

### CODE REVIEW

File: wg-multitenancy/README.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+Please do not edit this file directly, but instead make changes to the
+sigs.yaml file in the project root.
+
+To understand how this file is generated, see https://git.k8s.io/community/generator/README.md
+-->
+# Multitenancy Working Group
+
+Define the models of multitenancy that Kubernetes will support. Discuss and execute upon any remaining work that needs to be done to support these models. Create conformance tests that will prove that these models can be built and used in production environments.
+
+## Meetings
```
Comment (jessfraz) [inline wg-multitenancy/README.md]: I figure we can add meetings later if we need to :)

**Comment by castrojo** [comment]
For reference here's how to get set up for recording your meetings: https://github.com/kubernetes/community/blob/master/community/K8sYoutubeCollaboration.md

**Comment by cblecker** [comment]
/approve
/assign @davidopp 
for LGTM :)

**Pull request merged**